src/pip/_internal/commands/wheel.py

```diff
@@ -156,10 +157,12 @@ def run(self, options, args):
             reqs, check_supported_wheels=True
         )
 
-        reqs_to_build = [
-            r for r in requirement_set.requirements.values()
-            if should_build_for_wheel_command(r)
-        ]
+        reqs_to_build = []  # type: List[InstallRequirement]
+        for req in requirement_set.requirements.values():
+            if req.is_wheel:
+                preparer.save_linked_requirement(req, only_wheels=True)
```

Comment:
Doesn't this mean that the is_vcs check later will fail, making only_wheels into a no-op?